cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -221,20 +220,16 @@ public static String kafkaPodName(String cluster, int pod) {
         return kafkaClusterName(cluster) + "-" + pod;
     }
 
-    public static String clientsCASecretName(String cluster) {
-        return cluster + KafkaCluster.SECRET_CLIENTS_CA_SUFFIX;
+    public static String getClientsCaKeyName(String cluster) {
+        return cluster + KafkaCluster.SECRET_CLIENTS_CA_KEY_SUFFIX;
     }
 
     public static String brokersSecretName(String cluster) {
         return cluster + KafkaCluster.SECRET_BROKERS_SUFFIX;
     }
 
-    public static String clientsPublicKeyName(String cluster) {
-        return cluster + KafkaCluster.SECRET_CLIENTS_PUBLIC_KEY_SUFFIX;
-    }
-
-    public static String clusterPublicKeyName(String cluster) {
-        return getClusterCaName(cluster);
+    public static String getClientsCaName(String cluster) {
```

Comment:
The method name is a little confusing: At first glance it could be the name of the Secret, or the key within the Secret. I'm also not sure the `get` adds anything. It's not a JavaBean getter. What about `clientsCaCertSecretName()` and `clientsCaKeySecretName()`?